aea/runtime.py

```diff
@@ -100,12 +115,12 @@ def _stop(self) -> None:  # pragma: nocover
     @property
     def is_running(self) -> bool:
         """Get running state of the runtime."""
-        return self._state.get() == RuntimeStates.started
+        return self._state.get() == RuntimeStates.running
 
     @property
     def is_stopped(self) -> bool:
         """Get stopped state of the runtime."""
-        return self._state.get() in [RuntimeStates.stopped, RuntimeStates.initial]
+        return self._state.get() in [RuntimeStates.stopped]
```

Comment:
should we also have properties `is_starting` and `is_stopping`?

Reply:
we can add it if required.